style.css

```diff
@@ -0,0 +1,905 @@
+*{
+	box-sizing: border-box;
+}
+
+.container.fluid {
+    display: block;
+    background: rgb(0, 255, 242);
+    border: 1px solid brown;
+}
+
+.header-top {
+    background-color: #f72210;
+    color: white;
+    position: relative;
+}
+
+.header-top a {
+    color: white;
+    font-size: 16px;
+}
+
+.middle img {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    width: 55%; 
+}
+
+.page-header .date {
+    display: block;
+    margin-left: 10px;
+    margin-top: 35px;
+}
+
+.header-top .left-side {
+    font: bold 14px/14px "Roboto Condensed",sans-serif;
+    text-transform: uppercase;
+    justify-content: flex-end;
+    align-items: flex-end;
+}
+
+.header-top .left-side a {
+    text-decoration: none;
+    color: white;
+}
+
+.header-top .left-side .button {
+    background-color: white;
+    color: #f72210;
+}
+
+@media screen and (max-width: 990px) {
+.header-top .date { visibility: hidden; margin-top: 1px}
```

Comment:
#### `HINT`

:bulb: Let's make this `display: none` instead. Element with `visibility: hidden` occupied still occupy its space with the exception of not showing. Using  `visibility: hidden` will keep the logo from rendering on the far left of the page as in the source page.

```suggestion
.header-top .date { display: none; margin-top: 1px}
```